packages/dd-trace/src/remote_config/index.js

```diff
@@ -0,0 +1,257 @@
+'use strict'
+
+const uuid = require('crypto-randomuuid')
+const { EventEmitter } = require('events')
+const Scheduler = require('./scheduler')
+const tracerVersion = require('../../../../package.json').version
+const request = require('../exporters/common/request')
+const log = require('../log')
+
+const clientId = uuid()
+
+const POLL_INTERVAL = 5e3
+
+// There MUST NOT exist separate instances of RC clients in a tracer making separate ClientGetConfigsRequest
+// with their own separated Client.ClientState.
+class RemoteConfigManager extends EventEmitter {
+  constructor (config) {
+    super()
+
+    this.scheduler = new Scheduler((cb) => this.poll(cb), POLL_INTERVAL)
+
+    this.requestOptions = {
+      method: 'POST',
+      url: config.url,
+      hostname: config.hostname,
+      port: config.port,
+      // TODO: do we need hostname/port here ?
```

Comment:
// TODO: do we need hostname/port here ?

Reply:
Do we?

Reply:
Telemetry did include the three props for it to work with UDS so i'll do that too.